### PR TITLE
User defined signs

### PR DESCRIPTION
**Pull request by itsakifa**
new table user_sign

### CODE REVIEW

File: data_model/changelogs/0006/0006_user_signs.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
+--Add to signalo_db.sign a field named fk_user_sign
+ALTER TABLE signalo_db.sign 
+    ADD column fk_user_sign text;
+;
+
+-- Add the new sign type 'user defined' to the value list signalo_db.vl_sign_type
+INSERT INTO signalo_db.vl_sign_type (id, active, value_de, value_fr) VALUES (15, true, 'benutzerdefiniert', 'défini par l''utilisateur');
```
Comment (3nids) [inline data_model/changelogs/0006/0006_user_signs.sql:7]: tu rajoutes aussi l'italien?

Comment (itsakifa) [reply data_model/changelogs/0006/0006_user_signs.sql:7]: Normalement, la table n'a pas/plus la colonne value_it

Comment (3nids) [reply data_model/changelogs/0006/0006_user_signs.sql:7]: juste!

File: data_model/app/vw_sign_symbol.py [hunk 2]
```diff
@@ -35,61 +35,86 @@ def vw_sign_symbol(srid: int, pg_service: str = None):
...
-                  WHEN complex IS TRUE THEN 'complex.svg'
-                  WHEN fk_sign_type = 11 THEN vl_official_sign.img_de
-                  WHEN fk_sign_type = 12 THEN 'marker.svg'
-                  WHEN fk_sign_type = 13 AND mirror_red_frame IS TRUE THEN 'mirror.svg'
-                  WHEN fk_sign_type = 13 AND mirror_red_frame IS FALSE THEN 'mirror-noframe.svg'
-                  WHEN fk_sign_type = 14 THEN 'street-plate.svg'
-                END as _img_de
+                    WHEN sign.fk_sign_type = 15 THEN vl_user_sign.value_de
+                    ELSE vl_official_sign.value_de
+                  END AS _symbol_value_de
```
Comment (3nids) [inline data_model/app/vw_sign_symbol.py:41]: If we have these in the same field, this means than the path will be dynamically set depending on the sign_type, correct?

Comment (3nids) [reply data_model/app/vw_sign_symbol.py:41]: sorry, misread

**Pull request merged**